### ISSUE TITLE
[network] Confirm and get rid of keep-alives in Notifications protocol

### ISSUE THREAD

**Issue by dmitry-markin** [paritytech/polkadot-sdk#6350]
After https://github.com/paritytech/polkadot-sdk/pull/6248 is merged, manually keeping streams alive during [additional keep-alive timeout](https://github.com/paritytech/polkadot-sdk/blob/7903b117a7be8f02aabcd1e870608a4e2477b6b5/substrate/client/network/src/protocol/notifications/handler.rs#L727) should not be needed.

Confirm this and get rid of it.

**Comment by nazar-pc** [comment]
I believe this is what https://github.com/paritytech/polkadot-sdk/pull/6109 does, I specifically did `kademlia_legacy_protocol: _,` in https://github.com/paritytech/polkadot-sdk/pull/6248 to reduce inevitable merge conflict with it.

**Comment by dmitry-markin** [comment]
> I believe this is what #6109 does, I specifically did `kademlia_legacy_protocol: _,` in #6248 to reduce inevitable merge conflict with it.

This seems not related to Notifications protocol keep-alive timeouts for me.

**Comment by nazar-pc** [comment]
Indeed, I didn't read it carefully :man_facepalming:

**Comment by lexnv** [comment]
Indeed, believe we can remove it. Lets do it as a follow-up to not alter the started tests.


It looks like with this approach we are keeping the connection alive for at least 5 seconds of inactivity:
https://github.com/paritytech/polkadot-sdk/blob/7903b117a7be8f02aabcd1e870608a4e2477b6b5/substrate/client/network/src/protocol/notifications/handler.rs#L107
